src/d3d9/d3d9_common_texture.cpp

```diff
@@ -184,6 +184,14 @@ namespace dxvk {
     if (pDesc->MipLevels == 0 || pDesc->MipLevels > maxMipLevelCount)
       pDesc->MipLevels = maxMipLevelCount;
 
+    if (unlikely(pDesc->Discard)) {
+      if (!IsDepthStencilFormat(pDesc->Format))
+        return D3DERR_INVALIDCALL;
+
+      if (pDesc->Format == D3D9Format::D32_LOCKABLE || pDesc->Format == D3D9Format::D32F_LOCKABLE || pDesc->Format == D3D9Format::D16_LOCKABLE)
```

Comment:
Is the S8_LOCKABLE case left out intentionally here, or would it be possible to call IsLockableDepthStencilFormat here too?